addons/editor/stringtable.xml

```diff
@@ -393,6 +393,12 @@
         <Key ID="STR_ZEN_Editor_ForceFire_Description">
             <English>Makes selected AI units fire their current weapon while the key is held down. Vehicles will fire their first available turret with a weapon.</English>
         </Key>
+        <Key ID="STR_ZEN_Editor_ToggleLaser">
+            <English>Toggle Laser</English>
+        </Key>
+        <Key ID="STR_ZEN_Editor_ToggleLaser_Description">
+            <English>Makes selected AI units (vehicles) toggle their turret laser weapons on or off.</English>
```

Comment:
IMO clearer like this IIUC:
```suggestion
            <English>Makes selected AI units in vehicles toggle their turret laser weapons on or off.</English>
```
I was gonna suggest `AI vehicles` instead, but remembered that drones exist (UGV Stomper and MQ-12 Falcon FTW), which are internally using an AI unit cuz 'Arma' Engine.